exercises/random_number.cpp

```diff
@@ -11,23 +11,28 @@ using namespace std;
 
 #define EXIT_SUCCECS 0
 
+void input(int*max,int* min )
```

Comment:
non era necessario usare i puntatori, potevi scrivere:
```suggestion
void input(int &max,int &min )
```

senza dare in input il riferimento